Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -1227,8 +1229,16 @@ def point_data(p):
 
         act_pos = self.scatterplot_item.mapFromScene(event.scenePos())
         points = self.scatterplot_item.pointsAt(act_pos)
+
         if len(points):
-            text = "<hr/>".join(point_data(point) for point in points)
+            if len(points) > MAX_POINTS_IN_TOOLTIP:
+                text = "{} instances<hr/>{}<hr/>...".format(
+                    len(points),
+                    "<hr/>".join(point_data(point) for point in points[:5])
```

Comment:
Is the difference between the constant MAX_POINTS_IN_TOOLTIP above and the hard-coded 5 here intentional?

Reply:
yes. Displaying 20 points is ugly, but might be desired in some cases. When there are more, 5 are shown as a teaser. IMHO the tooltip looks much nicer this way :)